firmware/library/L1_Drivers/system_clock.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+
+class SystemClock
+{
+ public:    
```

Comment:
Method names should be captialized, see Google C++ style guide 
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules